plugins/auth-backend-module-aws-alb-provider/src/helpers.ts

```diff
@@ -70,6 +70,17 @@ export const makeProfileInfo = (
   };
 };
 
+const getPublicKeyEndpoint = (region: string) => {
+  const commercialEndpoint = `https://public-keys.auth.elb.${encodeURIComponent(
+    region,
+  )}.amazonaws.com/`;
+  const govEndpoint = `https://s3-${encodeURIComponent(
+    region,
+  )}.amazonaws.com/aws-elb-public-keys-prod-${encodeURIComponent(region)}/`;
+
+  return region.startsWith('us-gov') ? govEndpoint : commercialEndpoint;
```

Comment:
Could you structure this as `if (region.startsWith('us-gov')) { return ... }; return ...` instead? It's a bit confusing atm whether we actually need to construct both or not :p